server/controllers/auth/login.js

```diff
@@ -0,0 +1,34 @@
+const bcrypt = require('bcrypt');
```

Comment:
@ahmadysalah You can set your pull requests to target other branches that have their own pull requests.  So in this case, I think you created the `39-logout` branch from the `40-login` branch and so this pull request could target `40-login`.
Then this pull request would only have the logout-related changes and wouldn't also show the changes from `40-login`.  That makes it easier for the other reviewers and keeps your pull requests small and focused.
And then later you can always change this pull request's target branch back to `master`, once `40-login` has been merged to master itself. and everything should just work 😅

Reply:
I have this issue,
- so if you can help me, how I start a new job on another branch and already have a pull request, Waite, to review,

Reply:
Here is what I do, to work on multiple tasks:
Work on feature 1 (make a branch for it, for example `feature1`, from master)
Finish that and open a PR (`feature1` -> `master`)
Now start working on feature 2, which depends on feature 1, by making a new branch from the `feature1` branch.
Finish that, and open another PR *into* the feature1 branch (`feature2` -> `feature1`), so that the PR only shows the feature 2 changes, and not the whole set of feature 2 + feature 1 changes.

Then, once feature 1's PR is approved and merged to master, go into the feature 2 PR in github and edit it to now target master branch (`feature2` -> `master`).

Let me know if that makes sense or not.

Reply:
so, I have to create a new branch instead of another branch 
for example <login>  create <logout> and start new PR from the new one 

`Then, once feature 1's PR is approved and merged to master, go into the feature 2 PR in github and edit it to now target master branch (feature2 -> master).`

how can I do that from GitHub ..

Reply:
You don't have to change how you created the branches - you did it the right way already.

But you can change the PR's target branch (also it's sometimes called the "base branch" or "destination branch", it means the branch that you want to merge your changes into) in github as shown here:
https://i.stack.imgur.com/WHKrh.gif

Reply:
yes Ahmed , and as Tom said you can complete working on this branch as you need first branch work to complete in that .

Reply:
Yes exactly I was going to poat this
https://stackoverflow.com/questions/24159036/how-to-modify-a-pull-request-on-github-to-change-target-branch-to-merge-into

Reply:
thanks, for all of you 
i did it and its work :100: